src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -151,6 +156,14 @@ public void updateFilteredTaskList(Predicate<Task> predicate) {
         filteredTasks.setPredicate(predicate);
     }
 
+    @Override
+    public List<Project> getProjectList() {
+        List<Project> projectList = filteredTasks.stream().map(x -> x.getProject())
+                .filter(s -> StringUtils.isNotBlank(s.projectName))
```

Comment:
Do we need a third party library?

```suggestion
                .filter(s -> !s.projectName.isBlank())
```